src/basic/missing.h

```diff
@@ -1011,6 +1011,10 @@ typedef int32_t key_serial_t;
 #define ETHERTYPE_LLDP 0x88cc
 #endif
 
+#ifndef EXTRA_CMSG_SPACE
+#define EXTRA_CMSG_SPACE 1024
+#endif
```

Comment:
missing.h is for allow compilation against outdated glibc or kernel headers. EXTRA_CMSG_SPACE doesn't belong here.

Also, the `#ifndef` guard makes no sense for something that we define ourselved.